AllReadyApp/Web-App/AllReady.UnitTest/Controllers/CampaignControllerTests.cs

```diff
@@ -343,6 +410,26 @@ protected override string UrlEncode(string value)
             {
                 return urlEncodedValue;
             }
+        }
+
+        private class FakeUserManager : UserManager<ApplicationUser>
+        {
+            public FakeUserManager()
+                : base(new Mock<IUserStore<ApplicationUser>>().Object,
+                      new Mock<IOptions<IdentityOptions>>().Object,
+                      new Mock<IPasswordHasher<ApplicationUser>>().Object,
+                      new IUserValidator<ApplicationUser>[0],
+                      new IPasswordValidator<ApplicationUser>[0],
+                      new Mock<ILookupNormalizer>().Object,
+                      new Mock<IdentityErrorDescriber>().Object,
+                      new Mock<IServiceProvider>().Object,
+                      new Mock<ILogger<UserManager<ApplicationUser>>>().Object)
+            { }
+
+            public override Task<ApplicationUser> GetUserAsync(ClaimsPrincipal user)
+            {
+                return Task.FromResult(new ApplicationUser { Id = "123" });
+            }
         }
```

Comment:
do you need to use this class? I see it copied and pasted over a couple other test classes, and that's probably not a good thing. The GetUserAsync method is not even used by this controller test class.

Check out `AllReady.UnitTest.MockHelper.CreateUserManagerMock` and see if that gives you the functionality you need to test.

If not, opening a new Issue to centralize the `FakeUserManager` classes in:
ItineraryAdminControllerTests
CampaignControllerTests
UserAuthorizationServiceTests

might be worth it.